packages/react-components/src/templates/NewsAndEventsPageBody.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { ComponentProps } from 'react';
+
+import { NewsAndEventsCard } from '../organisms';
+
+interface LibraryPageBodyProps {
+  readonly newsAndEvents: ReadonlyArray<
+    ComponentProps<typeof NewsAndEventsCard>
+  >;
+}
+
+const NewsAndEventsPageBody: React.FC<LibraryPageBodyProps> = ({
+  newsAndEvents,
+}) => (
+  <>
```

Comment:
This will need to change slightly due to the `CardList` changes, sorry :/
Check e.g. `LibraryPageBody` - basically the cards are now wrapped in a `CardList`, and paddings/gridGaps that used to be in `NewsAndEventsPage` can also go away because they are now handled by `CardList`

Reply:
No worries, thanks for pointing it out.